commitizen/commands/bump.py

```diff
@@ -395,3 +411,33 @@ def _get_commit_args(self):
         if self.no_verify:
             commit_args.append("--no-verify")
         return " ".join(commit_args)
+
+    def find_previous_final_version(
```

Comment:
Good API addition 👍🏼

Reply:
We should probably rename the function to `find_base_version()` because “base version” seems to be the term used in other places.